api/lib/helpers/detainee-attributes.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var getPid = function getPid (head, tail) {
```

Comment:
better var names would make this easier to parse
you'd also benefit from using an arrow function so it could be rewritten:
```
var getPid = (centre, person_id) => `${centre}_${person_id}`
```